berkdb/rep/rep_record.c

```diff
@@ -654,6 +658,10 @@ __rep_process_message(dbenv, control, rec, eidp, ret_lsnp, commit_gen)
 			MUTEX_UNLOCK(dbenv, db_rep->rep_mutexp);
 		} else if (rp->rectype != REP_NEWMASTER) {
 
+			if (gbl_verbose_master_req) {
+				logmsg(LOGMSG_ERROR, "%s line %d sending REP_MASTER_REQ\n",
```

Comment:
Should be LOGMSG_USER (user requested this to be verbose, should always be emitted).  Minor nitpick.